src/util/widgetrendertimer.cpp

```diff
@@ -22,8 +24,16 @@ void WidgetRenderTimer::guiTick() {
 }
 
 void WidgetRenderTimer::activity() {
-    m_lastActivity = mixxx::Time::elapsed();
-    if (!m_guiTickTimer.isActive()) {
-        m_guiTickTimer.start(m_renderFrequency);
+#ifdef __APPLE__
+    if (m_openGlEnabled) {
+        m_lastActivity = mixxx::Time::elapsed();
+        if (!m_guiTickTimer.isActive()) {
+            m_guiTickTimer.start(m_renderFrequency);
+        }
+    } else {
+        emit update();
     }
+#else
+    emit update();
```

Comment:
This is new now. Is it really required?

Reply:
I tried commenting it out. Yes, it is required.

Reply:
What is the reason to change the code for other platforms than MacOS here?

Reply:
Otherwise the widgets do not update. Try it.

Reply:
I suppose I could #ifdef the whole `inputActivity` methods instead.

Reply:
Nope, that would require many more #ifdefs. I am not going down that rabbit hole.

Reply:
This is not really new. I only moved it into WidgetRenderTimer so each class that uses WidgetRenderTimer doesn't need to repeat the same #ifdefs.